streams/src/test/java/org/apache/kafka/streams/kstream/internals/KTableKTableLeftJoinTest.java

```diff
@@ -315,16 +316,37 @@ public void shouldNotThrowIllegalStateExceptionWhenMultiCacheEvictions() {
         final StreamsBuilder builder = new StreamsBuilder();
         final Consumed<Long, String> consumed = Consumed.with(Serdes.Long(), Serdes.String());
         final KTable<Long, String> aggTable = builder
-            .table(agg, consumed)
+            .table(agg, consumed, Materialized.as(Stores.inMemoryKeyValueStore("agg-base-store")))
```

Comment:
Huh. Bummer that you have to make it queriable in order to use the in-memory store. I never noticed that before.

Reply:
I think both tables at line 319 and line 321 are materialized anyways even without the `Materialized` object: for 319, the `groupBy` operator would cause its parent to be materialized.

Reply:
Yes. I only added `Materialized.as()` for the operators that materialized in any case (not not alter the test logic).